akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -200,6 +200,8 @@ on all nodes::
 You must explicitly add the ``akka-distributed-data-experimental`` dependency to your build if
 you use this mode. It is possible to remove ``akka-persistence`` dependency from a project if it
 is not used in user code and ``remember-entities`` is ``off``.
+Using it together with ``Remember Entities`` shards will be recreated after rebalancing, however will
```

Comment:
maybe add a small fragment to the end that the state is empty because ddata is purely in-memory (that drives the point home).